<!-- 启动所有语音节点的launch file-->
<launch>
	<!--
	<include file="$(find turtlebot_bringup)/launch/minimal.launch" />
    	<include file="$(find kamerider_image_detection)/launch/double_kinect.launch" />
    
	<include file="$(find kamerider_navigation)/launch/hmc_navigation.launch" />
	-->

	<!--
	<arg name="dict" default="/home/keaixin/catkin_ws/src/pocketsphinx/help-me-carry/2722.dic" />
	<arg name="lm"  default="/home/keaixin/catkin_ws/src/pocketsphinx/help-me-carry/2722.lm" />
	-->
	<arg name="dict" default="/home/zmx/catkin_ws/src/kamerider_speech/GPSR/6534.dic" />
	<arg name="kws"  default="/home/zmx/catkin_ws/src/kamerider_speech/GPSR/gpsr.kwlist" />
	
	<param name="is_restart" value="false" />
	<!-- 在help-me-carry中，应该在到达室外地点之后才开始等待命令 -->
	<param name="wait_for_command" value="false" />

	<!--
	<include file="$(find pocketsphinx)/launch/lm.launch">
		<arg name="dict" value="$(arg dict)" />
		<arg name="lm"  value="$(arg lm)" />
	</include>
	-->

	<include file="$(find pocketsphinx)/launch/kws.launch">
		<arg name="dict" value="$(arg dict)" />
		<arg name="kws"  value="$(arg kws)" />
	</include>

	<!--
	<include file="$(find turtlebot_follower)/launch/follower.launch">
	<include file="$(find kamerider_speech)/launch/hmc_follower.launch" >
	</include>
	-->
	
	<!-- 启动截取音频的节点
	<node pkg="kamerider_speech" type="get_audio_hmc.py" name="get_audio_hmc" launch-prefix="xterm -e" respawn="true" respawn_delay="1" />
	-->
	<!-- 启动科大讯飞识别节点	
	<node pkg="kamerider_speech" type="speech_recognition_hmc.py" name="speech_recognition_hmc" launch-prefix="xterm -e" respawn="true" respawn_delay="1" />
	-->
	<node pkg="xfei_asr" type="speech_recognition" name="speech_recognition" launch-prefix="xterm -e" respawn="true" respawn_delay="1" />

	<node pkg="sound_play" type="soundplay_node.py" name="sound_play" launch-prefix="xterm -e" />

	<node pkg="kamerider_speech" type="hmc_control.py" name="hmc_control" launch-prefix="xterm -e" />

	<!-- 设置摄像头参数 
    <include file="$(find turtlebot_bringup)/launch/3dsensor.launch">
    <arg name="rgb_processing" value="false" />
    <arg name="depth_registration" value="false" />
    <arg name="depth_processing" value="false" />
    <arg name="scan_topic" value="/scan" />
    <arg name="camera" value="kinect_up" />
    </include>-->

</launch>
